crypto-ffi/src/wasm/context/mod.rs

```diff
@@ -18,6 +18,8 @@ use wasm_bindgen::prelude::wasm_bindgen;
 use wasm_bindgen::JsValue;
 use wasm_bindgen_futures::future_to_promise;
 
+mod e2ei;
```

Comment:
Why is the the one from `generic` `pub`, and this one not?

Reply:
Good catch, made it consistent to be pub in both places.